### PR TITLE
Create basic cube sample

### LINKED ISSUE #57: Add "starter" sample
I think that there will be good idea to add "starter" sample, which should be an easy start with Tyra.

### PR DESCRIPTION
**Pull request by Wellinator**
### I hope you guys enjoy it!

This is a very basic sample that display a rotating cube on the screen. I'm still learning how tyra works.
Hare is the "assets" folder. [assets.zip](https://github.com/h4570/tyra/files/6292584/assets.zip) _(Cube Sample Assets)_

Fixes #57

### CODE REVIEW

File: src/samples/cube/camera.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/*
+# ______       ____   ___
+#   |     \/   ____| |___|    
+#   |     |   |   \  |   |       
+#-----------------------------------------------------------------------
+# Copyright 2020, tyra - https://github.com/h4570/tyra
+# Licenced under Apache License 2.0
+# Sandro Sobczyński <sandro.sobczynski@gmail.com>
```
Comment (h4570) [inline src/samples/cube/camera.cpp]: Please change it to your nickname in all of these comments :)

**Comment by h4570** [comment]
Hi @Wellinator.
Thank you for contribution!
I will check & run that code today or tommorow.
Thanks!

**Review by Wellinator — COMMENTED** [review]
Added file header

**Review by h4570 — APPROVED** [review]
Great!

At final step, please:
a) add `README.MD` - you can copy this file from other samples
b) add `.gitignore` file - also copy it from other samples

Here are the links for the `README.MD`:
Assets
http://apgcglz.cluster028.hosting.ovh.net/tyra/1.X.X/samples/cube/assets.zip
Gif
http://apgcglz.cluster028.hosting.ovh.net/tyra/1.X.X/samples/cube/cube.gif

**Comment by Wellinator** [comment]
README.MD added!

**Pull request merged**